Dockerfile

```diff
@@ -0,0 +1,89 @@
+# syntax=docker/dockerfile:1
+
+#	This Source Code Form is subject to the terms of the Mozilla Public License, 
+#	v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+#	obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under 
+#	the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#	
+#	Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS 
+#	graphic logo is a trademark of OpenMRS Inc.
+
+### Build Stage
+FROM maven:3.8-jdk-11 as build
+WORKDIR /app
+
+# Copy source
+COPY . .
+
+# Build the app using maven cache
+RUN --mount=type=cache,target=/root/.m2 mvn clean package -DskipTests
+
+### Run stage
+FROM tomcat:7-jdk8-adoptopenjdk-hotspot
+WORKDIR /app
+
+# All environment variables that are available to configure on this container are listed here
+# for clarity. These list the variables supported, and the default values if not overridden
+
+# These environment variables are appended to configure the Tomcat JAVA_OPTS
+ENV OMRS_JAVA_MEMORY_OPTS="-XX:NewSize=128m"
+ENV OMRS_JAVA_SERVER_OPTS="-Dfile.encoding=UTF-8 -server -Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -Djava.awt.headlesslib=true"
+
+# These environment variables are used to create the openmrs-server.properties file, which controls how OpenMRS initializes
+ENV OMRS_CONFIG_ADD_DEMO_DATA="false"
+ENV OMRS_CONFIG_ADMIN_USER_PASSWORD="Admin123"
+ENV OMRS_CONFIG_AUTO_UPDATE_DATABASE="true"
+ENV OMRS_CONFIG_CREATE_DATABASE_USER="false"
+ENV OMRS_CONFIG_CREATE_TABLES="false"
+ENV OMRS_CONFIG_HAS_CURRENT_OPENMRS_DATABASE="true"
+ENV OMRS_CONFIG_INSTALL_METHOD="auto"
+ENV OMRS_CONFIG_MODULE_WEB_ADMIN="true"
+
+# These variables are specific to database connections
+# Supported values for OMRS_CONFIG_CONNECTION_TYPE are "mysql" and "postgresql"
+# other values are treated as MySQL
+ENV OMRS_CONFIG_CONNECTION_TYPE="mysql"
+ENV OMRS_CONFIG_CONNECTION_USERNAME="openmrs"
+ENV OMRS_CONFIG_CONNECTION_PASSWORD="openmrs"
+ENV OMRS_CONFIG_CONNECTION_SERVER="localhost"
+ENV OMRS_CONFIG_CONNECTION_DATABASE="openmrs"
+
+# These environment variables can be used to customise the database connection.
+# Their default values depend on which database you are using.
+# OMRS_CONFIG_CONNECTION_DRIVER_CLASS
+# OMRS_CONFIG_CONNECTION_PORT
+# OMRS_CONFIG_CONNECTION_ARGS
+# OMRS_CONFIG_CONNECTION_EXTRA_ARGS
+#
+# If you really need complete control, you can just set
+# OMRS_CONFIG_CONNECTION_URL to whatever the URL should be
+
+# These environment variables are meant to enable developer settings
+# OMRS_DEV_DEBUG_PORT
+
```

Comment:
Not entirely sure why all of the above were commented out.  Are we assuming that no one should need to customize beyond the standard we are providing?